### PR TITLE
Feature/mongo

### PR DESCRIPTION
**Pull request by TekaLishoo**
Mongo + Schemas + Docker + Pagination

### CODE REVIEW

File: parsing_scripts/lamoda_parser.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    #
+    # consumer = KafkaConsumer(
+    #     'topic_test',
+    #     bootstrap_servers=['kafka:9092'],
+    #     auto_offset_reset='earliest',
+    #     enable_auto_commit=True,
+    #     group_id='my-group-id',
+    #     value_deserializer=lambda x: loads(x.decode('utf-8'))
+    # )
+    # for event in consumer:
```
Comment (achopik) [inline parsing_scripts/lamoda_parser.py]: Do not leave commented code, it's a bad practise

File: parsing_scripts/lamoda_parser.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    url = 'https://www.lamoda.by'
+    soup = await get_soup(url)
+    a_list = soup.find_all('a', attrs={'class': 'x-footer-seo-menu-tab-links__item'})
+    visited_products = set()
+    for a in a_list:
+        actual_url = url + a.attrs['href']
+        actual_soup = await get_soup(actual_url)
+        actual_a_list = actual_soup.find_all('a', attrs={'class': 'x-product-card__link'})
+        for i in actual_a_list:
+            print(f'{i.get_text()}')
```
Comment (achopik) [inline parsing_scripts/lamoda_parser.py]: 1. Make parser as a class, so you can use different parsers with the same interface. I would suggest such solution: 

```python
class AbstractParser(abc.ABC):
    
    BASE_URL: NotImplemented = None

    def __init__(self):
          pass # configuration goes here
    
    @abc.abstractmethod
    async def parse(self):
           pass # here comes parsing logic

class LamodaParser(AbstractParser):

     BASE_URL = "https://lamoda.by"

     async def parse(self):
         ... # actual implementation 
```
2. Don't use hardcoded in code properties such as `url`, use class attributes instead (see upper code)

**Review by achopik — CHANGES_REQUESTED** [review]

File: parsing_scripts/lamoda_parser.py (hunk 1 above)
Comment (TekaLishoo) [reply parsing_scripts/lamoda_parser.py]: Deleted

File: parsing_scripts/lamoda_parser.py (hunk 2 above)
Comment (TekaLishoo) [reply parsing_scripts/lamoda_parser.py]: Done as an example firstly.

**Pull request merged**